includes/wikia/models/NavigationModel.class.php

```diff
@@ -691,41 +660,42 @@ private function getBiggestCategory( $index ) {
 			$this->biggestCategories = WikiaDataAccess::cache(
 				wfMemcKey( 'biggest', $limit ),
 				604800, // a week
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)